src/pages/overlay/components/activationButtons/activationButtons.module.scss

```diff
@@ -1,16 +1,21 @@
 @import '../../../../variables.scss';
 
 .activationButtons{
-    margin-top: 40px;
+    margin-top: calc(6 * $overlay-base-size); //move slightly lower relative to available height
     button{
         all: unset;
+        display: flex;
+        width: calc(5 * $overlay-base-size); //scale based on available height
```

Comment:
can you make the size of the button bigger. it feels a bit too small.

Reply:
see https://github.com/abdullahmorrison/AlveusTwitchExtension/pull/26#discussion_r1088469243